gnat2goto/driver/tree_walk.adb

```diff
@@ -3091,19 +3091,27 @@ package body Tree_Walk is
          Lbound : constant Irep := Do_Expression (Low_Bound (Idx));
          Hbound : constant Irep := Do_Expression (High_Bound (Idx));
          Idx_Type : constant Entity_Id := Get_Array_Index_Type (E);
-         Len : constant Irep :=
-           Make_Array_Length_Expr (Lbound, Hbound, Idx_Type);
+         Source_Loc : constant Source_Ptr := Sloc (E);
          Component_Type : constant Irep :=
            Do_Type_Reference (Get_Array_Component_Type (E));
+         Len : constant Irep :=
```

Comment:
Why was this declaration moved (it seems to be identical to the one that was just slightly above this one before?)

Reply:
No particular reason, reverted.